pymc_marketing/clv/models/basic.py

```diff
@@ -103,6 +228,3 @@ def fit_summary(self, **kwargs):
             return res["mean"].rename("value")
         else:
             return az.summary(self.fit_result, **kwargs)
-
-    def __repr__(self):
```

Comment:
Does the base class have a `__repr__`? This method should actually have been `__str__` btw